### PR TITLE
Code review fix

### PR DESCRIPTION
**Pull request by opxz7148**
### Description

- Include `get_recent_activity` in code coverage #140 
- Remove duplicate method in Attend model class #144 
- Improve serializer class doctoring #147 
- Clarify class name to be more descriptive #148 
- Add missing step in google credentials guide

### Functional Acceptance Criteria
- Application running well after fix

### Checklist

- [x] I have performed a self-review of my code.
- [ ] I have added tests to cover my changes.
- [x] I have updated the documentation, if applicable.
- [x] My code follows the Coding Guidelines.

### Type of Change

- [ ] Bug fix
- [ ] New feature
- [x] Documentation update
- [x] Other (Improve code quality)

### CODE REVIEW

**Review by Thanawas-Sirilertsathit — APPROVED** [review]
You removed Code Defects. Approve.

File: activities/tests/test_activities.py
Comment (TeerapatTrepopsakulsin) [inline activities/tests/test_activities.py:1]: Please change the test method name and docstring according to changes

**Review by TeerapatTrepopsakulsin — COMMENTED** [review]
Great change according to code review checklist, don't for get to add these changes in [Follow-up Summary](https://docs.google.com/document/d/1j73MKVyNO99XD0evJh29MO1Q4HQZMtv6yWksD1PB0dY/edit?tab=t.v4hlxzk031yr), only minor things to modify and I'll approve this PR.

File: activities/tests/test_activities.py
Comment (opxz7148) [reply activities/tests/test_activities.py:1]: Fixed in lastest commit

**Review by TeerapatTrepopsakulsin — APPROVED** [review]
Commit reviewed. _Approve_

**Review by OmegaOoh — APPROVED** [review]
Well done, every class name are now descriptive.

**Pull request merged**